### DISCUSSION TITLE
SLE BCI based dotnet 8.0 container sets available

### DISCUSSION THREAD

**Discussion by dirkmueller** [SUSE/bci#16] [Announcements]
The fabulous SLE BCI team has just published the first set of dotnet 8.0 containers. Similar to dotnet 7.0 and 6.0, which continue to remain available, we have provided `bci/dotnet-sdk:8.0`, `bci/dotnet-aspnet:8.0` and `bci/dotnet-runtime:8.0` container and listed them on https://registry.suse.com/

**Comment by devpro** [comment]
When creating a new ASP.NET 8.0 project from the official template, I can see they changed the port numbers and added the USER statement at the beginning of the Dockerfile that is generated:

```Dockerfile
FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS base
USER app
WORKDIR /app
EXPOSE 8080
EXPOSE 8081
```

Should we follow the same standard for .NET BCI? (to ease the adoption)

**Comment by dirkmueller** [comment]
@devpro good point, will look into it.

**Reply by devpro** [reply]
Thank you very much. I did test the new 8.0 image, it works!

Exemple: [devpro/container-images/samples/dotnet/8.0](https://github.com/devpro/container-images/tree/main/samples/dotnet/8.0)
